HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -3816,7 +3819,7 @@ def from_oga(hpxml)
   end
 
   class LightingGroup < BaseElement
-    ATTRS = [:id, :location, :fration_of_units_in_location, :third_party_certification]
+    ATTRS = [:id, :location, :fraction_of_units_in_location, :third_party_certification, :lighting_type]
```

Comment:
Third party certification still gets used somewhere?

Reply:
Yes, it will still be the required input for ERI HPXML files.